.storybook/addons.js

```diff
@@ -1,2 +1,3 @@
 import '@storybook/addon-knobs/register';
 import '@storybook/addon-options/register';
+import '@storybook/addon-actions/register';
```

Comment:
what's the reason this was already added?

Reply:
I'll later mock insights with actions. 
It's like adding the insightsClient prop to the storybook previewWrapper while no story are using it yet: it's preparing for the next two PR (`hits with insights` and `infiniteHits with insights`) which will be using it